server/utils/recipeUtils.js

```diff
@@ -107,11 +82,26 @@ const resetPortions = asyncHandler(async (userId, Recipe) => {
 	}
 })
 
+const createNewRecipe = asyncHandler(async (recipeName, description, instructions, userID, Recipe) => {
+    await Recipe.sync()
+
+    return await Recipe.create({
+        recipe_name: recipeName,
+        description: description,
+        instructions: instructions,
+		selected: false,
+        user_id: userID,
+        imageUrl: "https://source.unsplash.com/random",
```

Comment:
should be string, not hardcode link here

Reply:
Would you prefer if I set imageUrl to be received from the front-end?

Reply:
yep should be like that

Reply:
Resolved.